Put SDF file writing back in correct place, fix mol_name issue, resolve #36

davidlmobley opened the pull request:
Resolves the mol_name error in #46 and simultaneously puts SDF file creation back in the correct place now that ParmEd no longer corrupts bond orders in `.mol2` files.

davidlmobley commented:
Oops, I forgot to merge this earlier. This means I'm also fixing #36 via the same PR, and this will form a new release (0.2.1). @bannanc , can you review the code changes relating to `solvated_mixtures.convert_to_gromacs` and make sure they look OK?

bannanc commented:
David,

Ok, I'm not sure if there is a way to do this on github, but here are some
comments, all minor:

1. Is there a reason for defining the wat_present variable? It seems like
you could just make line 204:

if 'WAT' in resnames:

and remove lines 199-201, but I tend to like not having extra variables if
I can avoid it.

Not related to your code, but my own curiosity:

1. I randomly noticed, you have Camila listed as a Solvation Toolkit author
(not on the solvated_mixtures part), since my logP paper is also
introducing Solvation Toolkit should she be on it?
2. Is there a reason to us <> instead of != when comparing two numbers?

Caitlin




*****************************************************
Caitlin C. Bannan
Graduate Student
Mobley Group
Chemistry Department at UC Irvine
bannanc@uci.edu


On Tue, Apr 19, 2016 at 4:15 PM, David L. Mobley <notifications@github.com>
wrote:

> Oops, I forgot to merge this earlier. This means I'm also fixing #36
> <https://github.com/MobleyLab/SolvationToolkit/issues/36> via the same
> PR, and this will form a new release (0.2.1). @bannanc
> <https://github.com/bannanc> , can you review the code changes relating
> to solvated_mixtures.convert_to_gromacs and make sure they look OK?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/MobleyLab/SolvationToolkit/pull/47#issuecomment-212167210>
>

bannanc commented:
by comments I meant comment, I typed that sentence before I finished
reading. It looks great :)



*****************************************************
Caitlin C. Bannan
Graduate Student
Mobley Group
Chemistry Department at UC Irvine
bannanc@uci.edu


On Tue, Apr 19, 2016 at 5:14 PM, Caitlin Colleen Bannan <bannanc@uci.edu>
wrote:

> David,
>
> Ok, I'm not sure if there is a way to do this on github, but here are some
> comments, all minor:
>
> 1. Is there a reason for defining the wat_present variable? It seems like
> you could just make line 204:
>
> if 'WAT' in resnames:
>
> and remove lines 199-201, but I tend to like not having extra variables if
> I can avoid it.
>
> Not related to your code, but my own curiosity:
>
> 1. I randomly noticed, you have Camila listed as a Solvation Toolkit
> author (not on the solvated_mixtures part), since my logP paper is also
> introducing Solvation Toolkit should she be on it?
> 2. Is there a reason to us <> instead of != when comparing two numbers?
>
> Caitlin
>
>
>
>
> *****************************************************
> Caitlin C. Bannan
> Graduate Student
> Mobley Group
> Chemistry Department at UC Irvine
> bannanc@uci.edu
>
>
> On Tue, Apr 19, 2016 at 4:15 PM, David L. Mobley <notifications@github.com
> > wrote:
>
>> Oops, I forgot to merge this earlier. This means I'm also fixing #36
>> <https://github.com/MobleyLab/SolvationToolkit/issues/36> via the same
>> PR, and this will form a new release (0.2.1). @bannanc
>> <https://github.com/bannanc> , can you review the code changes relating
>> to solvated_mixtures.convert_to_gromacs and make sure they look OK?
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/MobleyLab/SolvationToolkit/pull/47#issuecomment-212167210>
>>
>
>

davidlmobley commented:
>1. Is there a reason for defining the wat_present variable? It seems like
you could just make line 204:

The reason was that I thought I'd use it for something later but then didn't, and forgot I could simplify. :) Thanks, I've fixed this.

> I randomly noticed, you have Camila listed as a Solvation Toolkit author
(not on the solvated_mixtures part), since my logP paper is also
introducing Solvation Toolkit should she be on it?

She just contributed something the other day, so you weren't using her code. 

> Is there a reason to us <> instead of != when comparing two numbers?

So I had in my head python `<>` was the python 3 version, but I had it backwards - that's going away. So I just fixed this.

davidlmobley commented:
See, this is exactly why it's good to have someone else review your code. :)
